documentation/source/newsfragments/1629.feature.rst

```diff
@@ -0,0 +1,2 @@
+Only ``Makefile.sim`` is required to be included in user Makefile (``Makefile.inc`` is not needed).
+Add possibility to create custom simulator control by copying/creating simulator Makefiles and including instead of ``Makefile.sim``.
```

Comment:
I'm not sure if we really want to advertise/support this configuration, it could make it harder for us to change the build system in the future. I'd tend towards leaving this as a "hidden feature".

Reply:
Just mind that the only Makefile left in practice is `Makefile.inc` and with some luck will not change much.

Reply:
We are having so much work with slowly deprecating things currently around this subsystem, adding more API-like surface to it seems to me like a risk without much benefit.